### PR TITLE
feat: add possibility to `ignore` some errors by keywords

### PR DESCRIPTION
**Pull request by kseniyashaydurova**
https://saritasa.atlassian.net/browse/SD-789

Add possibility to ignore some errors by keywords. It is needed when nginx.conf contains `include` directives without real files presense in the repo (i.e. nginx defaults) - https://github.com/nginx/nginx/tree/master/conf

### CODE REVIEW

File: testing/no-try-files-with-default-nginx-includes/nginx.conf
Comment (dmitry-mightydevops) [inline testing/no-try-files-with-default-nginx-includes/nginx.conf:1]: maybe rename this folder into `tests`?

**Review by dmitry-mightydevops — APPROVED** [review]

File: testing/no-try-files-with-default-nginx-includes/nginx.conf
Comment (kseniyashaydurova) [reply testing/no-try-files-with-default-nginx-includes/nginx.conf:1]: We already have `tests` folder here - https://github.com/saritasa-nest/saritasa-pre-commit-hooks/tree/feature/ignore-some-errors/tests (it contains real pytests) and in `testing` folder there are different nginx configuration examples that are checked by tests from `tests` folder

Comment (dmitry-mightydevops) [reply testing/no-try-files-with-default-nginx-includes/nginx.conf:1]: tests/assets?

Comment (kseniyashaydurova) [reply testing/no-try-files-with-default-nginx-includes/nginx.conf:1]: done

**Pull request merged**